packages/api/src/controllers/asset.ts

```diff
@@ -527,7 +542,7 @@ app.post(
       asset
     );
 
-    res.json({ url, asset, task });
+    res.json({ url, asset, task: { id: task.id } });
```

Comment:
Hmm I feel like there are other places where we return tasks here. Shouldn't you be updating `transcode`, `request-upload` and `import` as well?